content/english/faq.md

```diff
@@ -60,6 +60,41 @@ Starting with HedgeDoc 2.0, mermaid is used for rendering sequence-diagrams. The
     ```
 {{< /faq-entry >}}
 
+{{< faq-entry title="Why do tags in headings don't work anymore?" draft=true >}}
+Starting with HedgeDoc 2.0, we don't support tag headings in the note anymore. 
+
+```
+#### tags: `tag1`, `tag2`
+```
+
+Instead please use the [frontmatter metadata](https://demo.hedgedoc.org/yaml-metadata#tags) to specify tags.
```

Comment:
With version 2.0 this URL will contain an additional /n/.

Reply:
good point